tools/interop-v2-reference.py

```
@@ -0,0 +1,726 @@
+
+from __future__ import annotations
+
+import importlib.util
+import json
+import sys
+from pathlib import Path
+
+_HERE = Path(__file__).resolve().parent
+
+
+def _load_v1():
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_load_v1`

(ANN202)